opm/autodiff/VFPHelpers.hpp

```diff
@@ -62,7 +62,9 @@ inline EvalWell zeroIfNanInf(const EvalWell& value) {
         OpmLog::warning("NAN_OR_INF_VFP_EVAL", "NAN or INF Evalution encountered during VFP calculation, the Evalution is set to zero");
     }
 
-    return nan_or_inf ? 0.0 : value;
+    using Toolbox = MathToolbox<EvalWell>;
+
+    return nan_or_inf ? Toolbox::createBlank(value) : value;
```

Comment:
Are you sure that createBlank() creates a zero? I thought it created an uninitialized value, not necessarily zero?

Reply:
It creates a Evaluation with everything to be zero.